cms/djangoapps/contentstore/views/course.py

```diff
@@ -1317,6 +1331,26 @@ def update_usage_info(store, course, configuration):
         configuration_json['usage'] = usage_information.get(configuration.id, [])
         return configuration_json
 
+    @staticmethod
+    def get_or_create_content_group_configuration(course):
+        """
+        Returns the first user partition from the course which uses the
+        CohortPartitionScheme, or generates one if no such partition is
+        found.  The created partition is not saved to the course until
+        the client explicitly creates a group within the partition and
+        POSTs back.
+        """
+        content_group_configuration = get_cohorted_user_partition(course.id)
+        if content_group_configuration is None:
+            content_group_configuration = UserPartition(
+                id=GroupConfiguration.generate_id(GroupConfiguration.get_used_ids(course)),
+                name='Content Group Configuration',
```

Comment:
I like this idea, though I'm a little overwhelmed by all the places we deal with group configurations, user partitions, and split tests in our code. Can I ask where a good place to put this constant would be?